### DISCUSSION TITLE
How to use it

### DISCUSSION THREAD

**Discussion by hxwe** [zwicker-group/py-pde#346] [General]
I am learing python for just one month.I still can not understand how to use the py-pde.But I have to solve a pde problem named Dual-temperature equation.So I need some advice on how to use it or where I can learn.

**Comment by david-zwicker** [comment]
We can only help you if you have a concrete question and demonstrated first attempts.

**Comment by hxwe** [comment]
Excuse me,I am still quite not sure whether it can solve heat conduction problems like this .
[heat conduction problems.docx](https://github.com/zwicker-group/py-pde/files/10213684/heat.conduction.problems.docx)
If so,how can I input the parameters.
Thanks!

**Comment by david-zwicker** [comment]
Having had a quick look at the document, I think this might be possible. Please read the documentation and examples carefully to learn how to use `py-pde`.
